CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 1.9.0 - 2022-11-16
+
+### Added
+
+- Create Function button in Workspace view [#3430](https://github.com/microsoft/vscode-azurefunctions/pull/3430)
```

Comment:
You should be listing the issues rather than the pull requests. The issues generally have more context and are easier to read.